src/com/lftechnology/java/training/dipak/employeemanagement/service/ServiceFactory.java

```diff
@@ -0,0 +1,21 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.service;
+
+/**
+ * <p>This class contains method to create object of the classes present in
+ * service layer.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class ServiceFactory {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)